packages/odyssey-react/src/components/Box/Box.theme.ts

```diff
@@ -13,16 +13,9 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  // REMOVE THIS
   /* eslint-disable @okta/odyssey/no-invalid-theme-properties */
```

Comment:
Required for the properties with `box` in them as well as properties that have no CSS equivalent like `MaxLineLength`.

Reply:
We'll want to apply this rule disable more surgically rather than at the top of entire theme module here (it applies all the way to the end of the file in this case)

Reply:
This is re-enabled below this group of properties. In this case, it applied wholly to groups, so I moved them into one escaped block rather than escape them individually.

Reply:
@edburyenegren-okta that works too! No strong feelings about how to organize tokens themselves within the reducer return object. I think we should feel free to alpha order when it makes sense, or group escaped rules like you're doing here!